Pages/about.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Intelligent Systems Club</title>
+  <title>Intelligent Systems Club/About</title>
```

Comment:
This is the text that shows on the tab I believe. I think it should stay "Intelligent Systems Club" no matter which page you're on.

https://www.w3schools.com/tags/tag_title.asp